lib/conditionals.md

```diff
@@ -5,8 +5,8 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A Conditional is an 'if then' statement that takes a program in a user defined direction.
 
 # What is the difference between `=` and `==`?
 
-Replace me with your response...
\ No newline at end of file
+'=' can define a variable within Ruby, while '==' is used to create a mathmatical equation to complete a code.
```

Comment:
`==` isn't always used for "mathematical" purposes. For example, if someone is trying to log into their account, you might use `==` to check that the password they are supplying matches the password they signed up with originally.

No reply or revision needed–just FYI.